### PR TITLE
Added paragraph into index.html file

### PR DESCRIPTION
**Pull request by amitkmr2396**
Made changes into the index.html file and added a paragraph into the file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,5 +5,6 @@
   <body>
     <h1> Welcome to the home page</h1>
     <h2> How you doing </h2>
+    <p> Adding content into the paragraph tag </p>
```
Comment (amitkmr404) [inline index.html:8]: Hurray...! , You made the nice change , Please also add some other tags

**Review by amitkmr404 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (amitkmr2396) [reply index.html:8]: Okay  thanks for confirmation ...

**Comment by amitkmr404** [comment]
Successfully merged the content of other aurthor

**Comment by amitkmr2396** [comment]
thanks a lot

**Comment by amitkmr404** [comment]
good bye

**Comment by amitkmr2396** [comment]
okey bye

**Pull request merged**